perception/radar_tracks_msgs_converter/schema/radar_tracks_msgs_converter.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Tracks Msgs Converter node",
+  "type": "object",
+  "definitions": {
+    "radar_tracks_msgs_converter": {
+      "type": "object",
+      "properties": {
+        "update_rate_hz": {
+          "type": "number",
+          "description": "The update rate [hz].",
+          "default": "20.0",
+          "minimum": 0.0
+        },
+        "new_frame_id": {
+          "type": "string",
+          "description": "The header frame of output topic.",
+          "default": "base_link"
```

Comment:
@PhoebeWu21 @mitsudome-r - I looked in `autoware.universe/perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp` to see if `new_frame_id` can be any value without success. Can this take any value?
If not, we could add an `enum` similar to as done in [refactor(joy_controller): rework parameters](https://github.com/autowarefoundation/autoware.universe/pull/3992/files#diff-a7768bad523a7c6d2571183d9240c85ca9829294291c048c8bec15df9b8ff5dfR13).

Reply:
Hi @kaspermeck-arm I think the value for `new_frame_id` can be assigned as any frame id name.